packages/client/src/Ethereum.ts

```diff
@@ -23,7 +23,7 @@ export const getStreamRegistryChainProviders = (config: Pick<StrictStreamrClient
 
 const getRpcProviders = (connectionInfo: ChainConnectionInfo, pollInterval?: number): Provider[] => {
     return connectionInfo.rpcs.map((c: ConnectionInfo) => {
-        const provider = new JsonRpcProvider(c)
+        const provider = new StaticJsonRpcProvider(c)
```

Comment:
Should we add a timeout while we are at it?
```suggestion
        const provider = new StaticJsonRpcProvider({
            ...c,
            timeout: 30 * 1000
        })
```

Reply:
In follow-up PR